library/constant_time_internal.h

```diff
@@ -129,6 +129,22 @@ unsigned mbedtls_ct_size_bool_eq( size_t x,
 unsigned mbedtls_ct_mpi_uint_lt( const mbedtls_mpi_uint x,
                                  const mbedtls_mpi_uint y );
 
+/**
+ * \brief          Check if an MPI is less than the other in constant time.
+ *
+ * \param X        The left-hand MPI. This must point to an array of limbs
+ *                 with the same allocated length as Y.
```

Comment:
don't we annotate arguments?

```suggestion
 *                 with the same allocated length as \p Y.
```